input/NVSS.yml

```diff
@@ -0,0 +1,10 @@
+mapper_class: MapperNVSS
+data_catalog: /mnt/extraspace/damonge/Datasets/NVSS/nvss.fits
+mask_sources: /mnt/extraspace/damonge/Datasets/NVSS/source_masks_nvss.txt
+redshift_catalog: /mnt/extraspace/damonge/Datasets/NVSS/100sqdeg_1uJy_s1400.fits
+DEC_min_deg: -40
+flux_min_mJy: 10
+fux_max_mJy: 1000
+GLAT_max_deg: 5
```

Comment:
Can we make this 10 instead of 5? Sorry, I know this is not what I used in the original yml

Reply:
Done!